src/data/staking-products.json

```
@@ -190,6 +190,33 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Vouch + Dirk go to link"
       }
+    },
+    {
+      "name": "AVADO",
+      "svgPath": "avado-glyph.svg",
+      "hue": 239,
+      "launchDate": "2018-12-01",
+      "url": "https://ava.do/",
+      "audits": [],
+      "minEth": 16,
+      "isFoss": true,
+      "hasBugBounty": false,
+      "isTrustless": true,
+      "isPermissionless": true,
+      "multiClient": true,
+      "easyClientSwitching": true,
+      "platforms": ["Linux", "macOS", "Windows", "Browser"],
```

My understanding is the Avado interface is accessed through a browser, and not as stand-along executable (similar to DAppNode above). Am I incorrect?
```suggestion
      "platforms": ["Browser"],
```